recommender/recommender.py

```diff
@@ -402,48 +401,48 @@ def _init_template_lookup(self):
             "resourcebox.html",
             self.resource_string("static/html/resourcebox.html"))
 
-    def get_client_side_settings(self):
+    def get_client_configuration(self):
         """
         Return the parameters for client-side configuration settings.
 
         Returns:
-                DISABLE_DEV_UX: feature flag for any new UX under development
+                disable_dev_ux: feature flag for any new UX under development
                                 which should not appear in prod
-                ENTRIES_PER_PAGE: the number of resources in each page
-                PAGE_SPAN: page range in pagination control
-                INTRO: whether to take users through a short usage tutorial
+                entries_per_page: the number of resources in each page
+                page_span: page range in pagination control
+                intro: whether to take users through a short usage tutorial
                        the first time they see the RecommenderXBlock
-                IS_USER_STAFF: whether the user is staff
+                is_user_staff: whether the user is staff
         """
-        result = self.client_side_settings.copy()
-        result['IS_USER_STAFF'] = self.get_user_is_staff()
-        result['INTRO'] = not self.seen and self.intro_enabled
+        result = self.client_configuration.copy()
+        result['is_user_staff'] = self.get_user_is_staff()
+        result['intro'] = not self.seen and self.intro_enabled
         if not self.seen:
             # Mark the user who interacted with the XBlock first time as seen,
             # in order not to show the usage tutorial in future.
             self.seen = True
-        tracker.emit('get_client_side_settings', result)
+        tracker.emit('get_client_configuration', result)
         return result
 
     @XBlock.json_handler
-    def set_client_side_settings(self, data, _suffix=''):  # pylint: disable=unused-argument
+    def set_client_configuration(self, data, _suffix=''):  # pylint: disable=unused-argument
         """
         Set the parameters for student-view, client side configurations.
 
         Args:
                 data: dict in JSON format. Keys in data:
-                  DISABLE_DEV_UX: feature flag for any new UX under development
+                  disable_dev_ux: feature flag for any new UX under development
                                   which should not appear in prod
-                  ENTRIES_PER_PAGE: the number of resources in each page
-                  PAGE_SPAN: page range in pagination control
-                  INTRO_ENABLE: Should we show the users a short usage tutorial
+                  entries_per_page: the number of resources in each page
+                  page_span: page range in pagination control
```

Comment:
Wouldn't `page_size` or `page_range` be a better name for this than `page_span`? At least, make it consistent between the name and the comment explaining what it is.